core/templates/pages/topic-editor-page/modal-templates/rearrange-skills-in-subtopics-modal.component.html

```diff
@@ -73,7 +73,7 @@ <h3>
             </div>
           </div>
           <div *ngIf="subtopic.getId() === selectedSubtopicId">
-            <input type="text" [(ngModel)]="editableName">
+            <input type="text" [(ngModel)]="editableName" maxlength="64">
```

Comment:
Why 64? Could you please explain for this ^, [this](https://github.com/oppia/oppia/pull/20081/files#diff-d73ca3bceeee4f9b56008923e0c7a206c800b6f307709a74a8a2f36cef75305dR51), [this](https://github.com/oppia/oppia/pull/20081/files#diff-d73ca3bceeee4f9b56008923e0c7a206c800b6f307709a74a8a2f36cef75305dR79) and [this](https://github.com/oppia/oppia/pull/20081/files#diff-21de6285412788cc0e1f25a079593ea7a1f14a1b835ce253154a59747c80c60dR55) why are you adding/changing the limit?

Reply:
While reassigning, we have the ability to update the names of subtopics. However, it’s important to note that the subtopic title is subject to a character limit of 64. This restriction is specified in the [assets/constants.ts](https://github.com/oppia/oppia/blob/develop/assets/constants.ts#L6200) file within the Oppia repository.

Alos, have addressed other concerns separately under respective lines.

Reply:
should I pass a variable instead?

Reply:
That sounds legitimate, and good that's you fix it, but it's not the best approach. When somebody updates that constant to another value this "64" value will not be updated. Use that constant pls.